docker-integrations/tendermint-python/.dockerignore

```diff
@@ -0,0 +1,9 @@
+**/node_modules/
```

Comment:
Python packages are not likely to have `node_modules` and other non-Python specific files and dirs. Maybe look at https://gist.github.com/KernelA/04b4d7691f28e264f72e76cfd724d448 which provides a more tailored `.dockerignore` for our purposes.

Reply:
Updated `.dockerignore` with the one proposed